velox/exec/fuzzer/CacheFuzzer.cpp

```diff
@@ -211,6 +225,25 @@ void CacheFuzzer::initSourceDataFiles() {
       fileSizes_.emplace_back(fileSize);
     }
   }
+
+  if (FLAGS_enable_file_faulty_injection) {
+    faultyFileSystem()->setFileInjectionHook([&](FaultFileOperation* op) {
+      if (std::regex_match(op->path, kDataFileNamePattern)) {
+        // Skip errors on input data files.
```

Comment:
What's the input data files here?

I think you could capture the input file directory here and match as prefix? Thanks!

Reply:
The input files are in the same directory, named as `file_{}` while cache files are named as `cache_{}`.